src/flatland/ordered/map.clj

```diff
@@ -86,7 +86,8 @@
     (.equiv this other))
   (hashCode [this]
     (reduce (fn [acc ^MapEntry e]
-              (let [k (.key e), v (.val e)]
+              (let [k (if-some [k (.key e)] k 0)
```

Comment:
Because `(= {nil nil} (ordered-map nil nil))` should be true, all clojure maps must use the same hashing strategy.